dotcom-rendering/src/components/Masthead/Masthead.tsx

```diff
@@ -82,7 +82,7 @@ export const Masthead = ({
 
 		{highlights && (
 			<Section
-				shouldCenter={false}
+				shouldCenter={true}
```

Comment:
Design have requested that we center the carousel using the standard whitespace padding